nimble/host/include/host/ble_gap.h

```diff
@@ -2167,6 +2194,44 @@ int ble_gap_set_prefered_default_le_phy(uint8_t tx_phys_mask,
 int ble_gap_set_prefered_le_phy(uint16_t conn_handle, uint8_t tx_phys_mask,
                                 uint8_t rx_phys_mask, uint16_t phy_opts);
 
+/**
+ * Set default subrate
+ *
+ * @param subrate_min       Min subrate factor allowed in request by a peripheral
+ * @param subrate_max       Max subrate factor allowed in request by a peripheral
+ * @param max_latency       Max peripheral latency allowed in units of
+ *                          subrated conn interval.
+ *
+ * @param cont_num  Min number of underlying conn event to remain active
+ *                          after a packet containing PDU with non-zero length field
+ *                          is sent or received in request by a peripheral.
+ *
+ * @param supervision_timeout Max supervision timeout allowed in request by a peripheral
+ */
+int ble_gap_set_default_subrate(uint16_t subrate_min, uint16_t subrate_max, uint16_t max_latency,
+                                uint16_t cont_num, uint16_t supervision_timeout);
+
+/**
+ * Set Subrate Request
+ *
+ * @param conn_handle       Connection Handle of the ACL.
+ * @param subrate_min       Min subrate factor allowed in request by a peripheral
```

Comment:
subrate request can be initiated by both central and peripheral so descriptions are not accurate (c&p...)

Reply:
Thank you. Have modified now.